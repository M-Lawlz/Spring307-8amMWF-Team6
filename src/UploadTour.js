import React from "react";
import AddTourForm from './AddTourForm'
import "firebase/firestore";

export default class UploadTour extends React.Component {

  constructor(props) {
    super(props);
    this.state = {tourId: "",
    userEmail: "",
    location: "",
    uploadDate: "",
    videoUrl: "",
    description: ""

  }
}


render() {
 return( 
  <div className="container">
  <h1>Upload Tours Here</h1>
  <div className="flex-row">
  <div className="flex-large">
  <h2>Add Tour</h2>
  <AddTourForm  />
  </div>
  <div className="flex-large">

  </div>
  </div>
  </div>
  )


}
}
