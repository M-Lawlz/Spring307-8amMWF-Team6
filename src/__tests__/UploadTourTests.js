import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UploadTour from "../UploadTour";

test("UploadTour renders successfully", () => {
  const uploadTour = shallow(<UploadTour />);
  expect(toJson(uploadTour)).toMatchSnapshot();
});
