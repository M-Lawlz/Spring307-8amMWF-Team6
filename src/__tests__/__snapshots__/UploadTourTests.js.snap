// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadTour renders successfully 1`] = `
<div
  className="container"
>
  <h1>
    Upload Tours Here
  </h1>
  <div
    className="flex-row"
  >
    <div
      className="flex-large"
    >
      <h2>
        Add Tour
      </h2>
      <AddTourForm />
    </div>
    <div
      className="flex-large"
    />
  </div>
</div>
`;
